src/Donations/Components/ContactsForm.tsx

```diff
@@ -275,9 +289,14 @@ function ContactsForm(): ReactElement {
               name="email"
               control={control}
               rules={{
-                required: true,
-                pattern:
-                  /^([a-z0-9!#$%&'*+/=?^_`{|}~-]+(?:\.[a-z0-9!#$%&'*+/=?^_`{|}~-]+)*@(?:[a-z0-9](?:[a-z0-9-]*[a-z0-9])?\.)+[a-z0-9](?:[a-z0-9-]*[a-z0-9])?)$/i,
+                required: {
+                  value: true,
+                  message: t("emailRequired"),
+                },
+                validate: {
+                  emailInvalid: (value) =>
+                    value.length === 0 || isEmailValid(value),
+                },
```

Comment:
How is the invalid email state shown? The code below to show errors seems to be written to exclude validate type errors, and currently shows the same "Email is required" message - this is because `errors.email.type === 'emailInvalid'` in case of an invalid email address. I'd suggest a different message when email is invalid as opposed to not provided at all. 

```
{errors.email && errors.email.type !== "validate" && (
  <span className={"form-errors"}>{t("emailRequired")}</span>
)}
```